### DISCUSSION TITLE
What happens if new message is sent to the account while my whatsmeow connection is disconnected

### DISCUSSION THREAD

**Discussion by llai-utexas** [tulir/whatsmeow#536] [WhatsApp protocol Q&A]
What happens if new message is sent to the account while my whatsmeow connection is disconnected. I wonder if the message received during this time would disappear, or would be received when the connection is online again

**Comment by mikhikhi** [comment]
as my experience, because your whatsmeow client play role as a another device , when you online again your history synced and you received that messages, i believed.
